.circleci/config.pkl

```diff
@@ -23,7 +23,9 @@ jobs {
       "checkout"
       new RunStep {
         command = """
-          curl -L -o pkl.bin https://github.com/apple/pkl/releases/download/0.25.1/pkl-linux-amd64
+          apt-get -y update
```

Comment:
5.9 is the newest one, but does not come with curl pre-installed :(

Reply:
Running on my local docker I needed it because the package cache comes empty and needs to be populated. Just calling `apt-get install` doesn't find curl.